Project.toml

```
@@ -44,6 +45,7 @@ HIP_jll = "4, 5"
 LLD_jll = "12, 13, 14"
 LLVM = "4.6"
 LLVM_jll = "12, 13, 14"
+MIOpen_jll = "5"
```

```suggestion
MIOpen_jll = "4, 5"
```
And an associated JLL needs to exist, otherwise Julia 1.7 won't be able to load AMDGPU due to resolver errors.

---

I'll look into adding MIOpen for 4.2.0 if it is not causing too many issues.